website/docs/r/scc_profile.html.markdown

```diff
@@ -10,7 +10,7 @@ subcategory: "Security and Compliance Center"
 
 Create, update, and delete profiles with this resource.
 
-~> NOTE: if you specify the `region` in the provider, that region will become the default URL. Else, exporting the environmental variable IBMCLOUD_SCC_API_ENDPOINT will override any URL(ex. `export IBMCLOUD_SCC_API_ENDPOINT=https://us-south.compliance.ibm.com`).
+~> NOTE: if you specify the `region` in the provider, that region will become the default URL. Else, exporting the environmental variable IBMCLOUD_SCC_API_ENDPOINT will override any URL(ex. `export IBMCLOUD_SCC_API_ENDPOINT=https://us-south.compliance.cloud.ibm.com`).
```

Comment:
In the resource we marked controls from required to optional. Can we update the this docuemnt also to reflect same

Reply:
the resource field `Controls` is changed back from required to optional